Sources/Public/View+.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  File.swift
```

Comment:
- Maybe we should reflect the file name.
- It might be better to add some words after the file name `View+`. For example,  "View+prettyPrint.swift".

Reply:
Thanks, I’ve changed it. [c3f3f99](https://github.com/YusukeHosonuma/SwiftPrettyPrint/pull/176/commits/c3f3f990ebb04610dc1cdb40cea19edf811fb9d9)